automatic/cutepdf/tools/chocolateyInstall.ps1

```diff
@@ -1,16 +1,30 @@
+﻿$ErrorActionPreference = 'Stop'
+
 $checksum = '2D9E07F25358C9D2317BC639AFCDEDDB893D1FCFD43BB66FF372DBA11E169EE1'
 $url = 'http://www.cutepdf.com/download/CuteWriter.exe'
 
 $packageArgs = @{
   packageName   = 'cutepdf'
-  unzipLocation = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
   fileType      = 'exe'
   url           = $url
   silentArgs    = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-' # Inno Setup Package
   validExitCodes= @(0)
-  softwareName  = 'cutepdf*'
   checksum      = $checksum
   checksumType  = 'sha256'
 }
 
+# Make sure Print Spooler service is up and running
+try {
+  $serviceName = 'Spooler'
+  $spoolerService = Get-WmiObject -Class Win32_Service -Property StartMode,State -Filter "Name='$serviceName'"
+  if ($spoolerService -eq $null) { throw "Service $serviceName was not found" }
+  Write-Host "Print Spooler service state: $($spoolerService.StartMode) / $($spoolerService.State)"
+  if ($spoolerService.StartMode -ne 'Auto' -or $spoolerService.State -ne 'Running') {
+    Set-Service $serviceName -StartupType Automatic -Status Running
```

Comment:
ah, I misunderstood the reason why it was being enabled.
I thought perhaps it just needed to be enabled to install, not to run.

Let me check out a few things regarding cutepdf before I merge this.